ui/src/app/views/table.html

```diff
@@ -1,12 +1,14 @@
 <div class="table-responsive-vertical md-whiteframe-z1">
-    <table id="table" class="table table-hover table-bordered">
+    <md-button class="md-primary md-raised" layout-align="center end" ng-click="changeView()"><i class="material-icons ng-scope" id="grid-button">view_module</i></md-button>
+    <table id="table" class="table table-hover table-bordered" ng-show="!isGridOn">
```

Comment:
might be a better name `isGridVisible`. @malithsen @agentmilindu  WDYT?

Reply:
Agree

Reply:
yes, correct. Let me refactor the code 😊